<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PILS Data Visualizer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="btn_frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>680</y>
      <width>1021</width>
      <height>80</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>1011</width>
       <height>30</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="selectFolder_label">
        <property name="text">
         <string>Select PILS folder path:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="directoryPath_lineEdit"/>
      </item>
      <item>
       <widget class="QToolButton" name="selectFolder_toolButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="start_btn">
        <property name="text">
         <string>Start Monitoring</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stop_btn">
        <property name="text">
         <string>Stop Monitoring</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="plot_btn">
        <property name="text">
         <string>Plot Selected File </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="export_btn">
        <property name="text">
         <string>Export as xlsx</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clear_btn">
        <property name="text">
         <string>Clear Plot</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="folder_tree_frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>271</width>
      <height>671</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>261</width>
       <height>661</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="files_label">
        <property name="text">
         <string>Files:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTreeView" name="folder_tree"/>
      </item>
      <item>
       <widget class="QLineEdit" name="status_lineEdit"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="plot_frame">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>761</width>
      <height>661</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>741</width>
       <height>641</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="PlotWidget" name="plot_tip_temp" native="true"/>
      </item>
      <item>
       <widget class="PlotWidget" name="plot_steam_temp" native="true"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Export As xlsx</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSelect_Data_Repository">
   <property name="text">
    <string>Select Data Repository</string>
   </property>
  </action>
  <action name="actionSelect_Data_Directory">
   <property name="text">
    <string>Select Folder</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header location="global">pyqtgraph.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
